fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/DropForeignKeyConstraint.java

```diff
@@ -42,8 +42,7 @@ public void run(IDatabaseTranslator translator, Connection c) {
 
         for (String constraintName : constraintNames) {
             StringBuilder ddl = new StringBuilder("ALTER TABLE " + qTableName);
-            String qConstraintName = DataDefinitionUtil.getQualifiedName(schemaName, constraintName);
-            ddl.append("\n\t" + "DROP FOREIGN KEY " + qConstraintName);
+            ddl.append("\n\t" + "DROP FOREIGN KEY " + constraintName);
```

Comment:
I think the prior line 45 was good defensive programming
it asserts if the passed schemaName and constraintName are valid

Reply:
It was computing the qualified name which we don't want.  I will add `DataDefinitionUtil.assertValidName(constraintName)` in a loop in the constructor for the input validation.

Reply:
done